src/components/sound.js

```diff
@@ -104,6 +104,12 @@ module.exports.Component = registerComponent('sound', {
       });
     }
 
+    sound.source.onended = function () {
+      // we've "overridden" sound's onended binding so call it here
+      sound.onEnded();
+      el.emit('onended');
```

Comment:
Can also add this event to the sound docs https://github.com/aframevr/aframe/blob/dev/docs/components/sound.md#properties

Similar to https://github.com/aframevr/aframe/blob/dev/docs/components/material.md#events